course/include/isometry.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+
+#include <initializer_list>
+#include <string>
+#include <sstream>
+#include <cmath>
+#include <iostream>
+
+namespace ekumen {
+  namespace math {
+
+class Vector3 {
+ public:
+  static const Vector3 kUnitX;
+  static const Vector3 kUnitY;
+  static const Vector3 kUnitZ;
+  static const Vector3 kZero;
+
+  Vector3(double x, double y, double z): x_(x), y_(y), z_(z) {};
+  Vector3(std::initializer_list<double> row): x_(row.begin()[0]), y_(row.begin()[1]), z_(row.begin()[2]) {};
+  Vector3(const Vector3 & obj): x_(obj[0]), y_(obj[1]), z_(obj[2]) {};
+  Vector3(): x_(0.), y_(0.), z_(0.) {};
+
+  const double & x() const {return x_;};
+  const double & y() const {return y_;};
+  const double & z() const {return z_;};
+  double & x() {return x_;};
+  double & y() {return y_;};
+  double & z() {return z_;};
+
+  const double & operator [] (const int index) const;
+  double & operator [] (const int index);
+  Vector3 operator += (const Vector3 & obj) const;
+  Vector3 operator -= (const Vector3 & obj) const;
+  Vector3 operator *= (const Vector3 & obj) const;
+  Vector3 operator /= (const Vector3 & obj) const;
```

Comment:
None of these operators with assignment can be `const`. Why?

Reply:
A const object cant use =/ or any of those, as a const cant be modified, and =/ modifiy the object